[2023-01-04 04:09:20,849] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-21T07:00:00+00:00 [queued]>
[2023-01-04 04:09:20,856] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-21T07:00:00+00:00 [queued]>
[2023-01-04 04:09:20,856] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 04:09:20,856] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-04 04:09:20,856] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 04:09:20,883] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_crypto_values> on 2021-02-21 07:00:00+00:00
[2023-01-04 04:09:20,888] {standard_task_runner.py:52} INFO - Started process 61362 to run task
[2023-01-04 04:09:20,906] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'populate_crypto_values', 'get_crypto_values', 'scheduled__2021-02-21T07:00:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/populate_crypto_values.py', '--cfg-path', '/tmp/tmpkd__7h2o', '--error-file', '/tmp/tmprua6szlx']
[2023-01-04 04:09:20,907] {standard_task_runner.py:80} INFO - Job 80: Subtask get_crypto_values
[2023-01-04 04:09:21,160] {task_command.py:370} INFO - Running <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-21T07:00:00+00:00 [running]> on host xuxon-virtual-machine
[2023-01-04 04:09:21,222] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=populate_crypto_values
AIRFLOW_CTX_TASK_ID=get_crypto_values
AIRFLOW_CTX_EXECUTION_DATE=2021-02-21T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-21T07:00:00+00:00
[2023-01-04 04:09:22,911] {logging_mixin.py:115} INFO - Encountered errors while inserting rows: [{'index': 0, 'errors': [{'reason': 'invalid', 'location': 'snapshot_date', 'debugInfo': '', 'message': "Invalid date: '22-02-2021'"}]}, {'index': 1, 'errors': [{'reason': 'invalid', 'location': 'snapshot_date', 'debugInfo': '', 'message': "Invalid date: '22-02-2021'"}]}]
[2023-01-04 04:09:22,911] {python.py:173} INFO - Done. Returned value was: None
[2023-01-04 04:09:22,921] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=populate_crypto_values, task_id=get_crypto_values, execution_date=20210221T070000, start_date=20230104T070920, end_date=20230104T070922
[2023-01-04 04:09:22,944] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-04 04:09:22,967] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-01-04 05:24:03,353] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-21T07:00:00+00:00 [queued]>
[2023-01-04 05:24:03,403] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-21T07:00:00+00:00 [queued]>
[2023-01-04 05:24:03,403] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 05:24:03,403] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-04 05:24:03,403] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 05:24:03,421] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_crypto_values> on 2021-02-21 07:00:00+00:00
[2023-01-04 05:24:03,426] {standard_task_runner.py:52} INFO - Started process 8671 to run task
[2023-01-04 05:24:03,436] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'populate_crypto_values', 'get_crypto_values', 'scheduled__2021-02-21T07:00:00+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/populate_crypto_values.py', '--cfg-path', '/tmp/tmprql4e7xi', '--error-file', '/tmp/tmpve9g3gkn']
[2023-01-04 05:24:03,437] {standard_task_runner.py:80} INFO - Job 109: Subtask get_crypto_values
[2023-01-04 05:24:03,550] {task_command.py:370} INFO - Running <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-02-21T07:00:00+00:00 [running]> on host xuxon-virtual-machine
[2023-01-04 05:24:03,696] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=populate_crypto_values
AIRFLOW_CTX_TASK_ID=get_crypto_values
AIRFLOW_CTX_EXECUTION_DATE=2021-02-21T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-21T07:00:00+00:00
[2023-01-04 05:24:05,361] {logging_mixin.py:115} INFO - New rows have been added.
[2023-01-04 05:24:05,361] {python.py:173} INFO - Done. Returned value was: None
[2023-01-04 05:24:05,370] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=populate_crypto_values, task_id=get_crypto_values, execution_date=20210221T070000, start_date=20230104T082403, end_date=20230104T082405
[2023-01-04 05:24:05,394] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-04 05:24:05,416] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
